### ISSUE TITLE
Button to register eyepiece and camera

### ISSUE THREAD

**Issue by brandon-gallas** [DIDSR/eeDAP#43]
What do you think about adding a button to the data collection gui that allows the user to do the eyepiece/camera registration?

**Comment by qigongFDA** [comment]
Yes. We can do that. But I want to think a little more about the meaning of it.
